composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/CalendarItem.kt

```diff
@@ -0,0 +1,228 @@
+package com.wisemen.chargehub.ui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ChevronRight
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.wisemen.chargehub.R
+import com.wisemen.chargehub.SharedRes
+import com.wisemen.chargehub.toStringResource
+import com.wisemen.chargehub.ui.theme.AppTheme
+import com.wisemen.chargehub.ui.theme.Colors
+import com.wisemen.chargehub.ui.theme.TextStyles
+
+@Preview
+@Composable
+fun CalenderItemsPreview() {
+    AppTheme {
+        Column {
+            CalenderItem(
+                "John Doe",
+                "Tesla",
+                painterResource(id = R.drawable.placeholder_profile),
+                1
+            )
+            CalenderItemPersonal(
+                "Cupra",
+                painterResource(id = R.drawable.placeholder_profile),
+                2
+            )
+            CalenderItem(
+                "Jane Doe",
+                "Tesla",
+                painterResource(id = R.drawable.placeholder_profile),
+                3
+            )
+        }
+    }
+}
+
+@Composable
+fun CalenderItemPersonal(
```

Comment:
First of all, don't work with these hardcoded values for every height or size. As I suggested, try to create a ratio between px or dp and duration and convert it that way. This is not maintainable and/or clear what all these values represent and why.